src/site/content/en/handbook/content-types/index.md

```diff
@@ -9,20 +9,20 @@ description: |
 ---
 
```

Comment:
Do people decide what the content type is? There should be an intro paragraph to set up this page that explains what content types are, why we have them on web.dev, and that you should choose one when you're submitting a proposal, etc.